src/content/advanced/spot-instances/azure/ondemand-fallback/index.md

```diff
@@ -0,0 +1,174 @@
+
+To ensure enough capacity in a cluster, this guide presents a solution where
+
+- two node pools exist: one configured to only use spot instances, the other one to only use on-demand instances.
+- cluster-autoscaler is configured to scale up the spot node pool first, and only if that fails, scale up the on-demand node pool.
+
+Credit where credit is due: this solution has been proposed [in a blog post by Nir Forer](https://blog.doit-intl.com/running-eks-workloads-on-spot-instances-with-on-demand-instances-fallback-14bef39ce689). We adapt it here to match exactly the situation of Giant Swarm workload clusters on Azure.
+
+**Note:** This guide has been written for workload cluster release v14.1.0 with [cluster-autoscaler](https://github.com/kubernetes/autoscaler) v1.19.1, provided via our [cluster-autoscaler-app](https://github.com/giantswarm/cluster-autoscaler-app) version [v1.19.1](https://github.com/giantswarm/cluster-autoscaler-app/releases/tag/v1.19.1).
+
+## Cluster set-up
+
+First, create a cluster with at least version 14.1.x for the purpose of this tutorial.
+
+Next, create two node pools.
+
+1. Name: Spot,
+   Enable spot instances: yes,
+   Spot maximum price per hour: set your value for max price or use current on-demand pricing as max,
+   Scaling range: min=1
+
+2. Name: On-Demand,
+   Instance type: the same as selected for the "Spot" node pool,
+   Enable spot instances: no,
+   Scaling range: min=1
+
+If it is important to you that the nodes in both node pools run in the same availability zone(s), you have to select the matching zones manually for each node pool.
+
+Make sure there are no other node pools in the cluster.
+
+Finally, once you have create the cluster and the two node pools, take note of their IDs. You will need them in the next steps.
+
+## Autoscaler expander configuration
+
+First we create a ConfigMap manifest to provide configuration for the cluster-autoscaler's `priority` expander. This expander decides which node pool to scale up, based on a defined priority value.
+
+Let's assume that our Spot node pool has the ID `sp0ti` and the on-demand node pool has ID `0ndmd`. We apply these two node pool ID in the ConfigMap as shown below:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-autoscaler-priority-expander
+  namespace: kube-system
+data:
+  priorities: |-
+    10:
+      - .*0ndmd.*
+    20:
+      - .*sp0ti.*
+```
+
+The `.data.priorities` part defines a map of priority values and regular expressions to select the Auto Scaling Group (ASG) associated with the node pool. Since with Giant Swarm the node pool ID is part of the respective Virtual Machine Scale Set (VMSS) name, we have to apply the IDs here.
```

Comment:
```suggestion
The `.data.priorities` part defines a map of priority values and regular expressions to select the Virtual Machine Scale Set (VMSS) associated with the node pool. Since with Giant Swarm the node pool ID is part of the respective VMSS name, we have to apply the IDs here.
```